hal/armv7a/_hal.S

```diff
@@ -0,0 +1,220 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * Low-level hal functions for Cortex-A9
+ *
+ * Copyright 2018, 2020, 2021 Phoenix Systems
+ * Author: Pawel Pisarczyk, Aleksander Kaminski, Maciej Purski, Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
```

Comment:
We need to make sure that this file is compiled to ARM asm. Rest of the kernel should be in thumb2. So we need .arm directive here

Reply:
Done